daemon/graphdriver/driver.go

```diff
@@ -173,9 +199,19 @@ func New(root string, options []string) (driver Driver, err error) {
 	return nil, fmt.Errorf("No supported storage backend found")
 }
 
+func scanPriorDrivers(root string) []string {
+	priorDrivers := []string{}
```

Comment:
I don't think I'll do a map here, because that would be unordered and still require the comparison. Perhaps I can just revisit the double iteration